### PR TITLE
Refactor advanced search filter tags, refs #12815

### PR DESCRIPTION
**Pull request by mcantelon**
Refactored filter tags so all of them can be rendered by a component.

Filter tag configuration can be added to an advanced search action
class and, with a call to a method of the parent DefaultBrowseAction
class, a single partial uses to display all filter tags.

### CODE REVIEW

**Comment by mcantelon** [comment]
(After merging this I'll refactor other advance search actions using the filter tags partial.)

File: apps/qubit/modules/actor/templates/browseSuccess.php [hunk 1]
```diff
@@ -68,14 +68,14 @@
         <?php echo get_partial('search/aggregation', array(
           'id' => '#facet-places',
           'label' => sfConfig::get('app_ui_label_place'),
-          'name' => 'places',
+          'name' => 'place',
```
Comment (jraddaoui) [inline apps/qubit/modules/actor/templates/browseSuccess.php:71]: Why is this changed?

File: apps/qubit/modules/actor/templates/browseSuccess.php [hunk 2]
```diff
@@ -68,14 +68,14 @@
...
           'aggs' => $aggs,
           'filters' => $search->filters)) ?>
 
         <?php echo get_partial('search/aggregation', array(
           'id' => '#facet-subjects',
           'label' => sfConfig::get('app_ui_label_subject'),
-          'name' => 'subjects',
+          'name' => 'subject',
```
Comment (jraddaoui) [inline apps/qubit/modules/actor/templates/browseSuccess.php:78]: And this one?

File: apps/qubit/modules/actor/actions/browseAction.class.php [hunk 3]
```diff
@@ -216,46 +227,21 @@ protected function populateAgg($name, $buckets)
...
-    {
-      $this->subject = QubitTerm::getById($request->subject);
-    }
-
-    if (isset($request->mediatypes) && ctype_digit($request->mediatypes))
-    {
-      $this->mediatypes = QubitTerm::getById($request->mediatypes);
-    }
+    // Set label for empty field filter tag
+    $labelText = $this->i18n->__('Empty: %1%', array('%1%' => $this->fieldOptions[$request->emptyField]));
```
Comment (jraddaoui) [inline apps/qubit/modules/actor/actions/browseAction.class.php]: Should we check if `$request->emptyField` is set?

**Review by jraddaoui — COMMENTED** [review]
Really nice @mcantelon! Just a few comments. Thanks!!

File: apps/qubit/modules/actor/templates/browseSuccess.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/actor/templates/browseSuccess.php:71]: This facet isn't currently working in qa/2.5.x so I think I misnamed it at some point. Same with subject.

File: apps/qubit/modules/actor/actions/browseAction.class.php (hunk 3 above)
Comment (mcantelon) [reply apps/qubit/modules/actor/actions/browseAction.class.php]: Good call.

**Comment by mcantelon** [comment]
I made that change... ready for another review.

**Review by jraddaoui — APPROVED** [review]
